.mergify.yml

```diff
@@ -9,11 +9,15 @@ pull_request_rules:
       - label!=ci-waived
       - or:
         # Build test
-        - -check-success=build (gcc, 11, bfd, gcrypt)
-        - -check-success=build (gcc, 12, gold, openssl)
-        - -check-success=build (clang, 13, mold, gcrypt) 
-        - -check-success=build (clang, 14, lld, openssl)
-        - -check-success=build (clang, 15, bfd, auto)
+        # test build check only when specific files are changed ...
+        - and:
+          - files~=^(\S+\/meson\.build|\.github\/workflows\/\S+|meson_options\.txt|src\/\S+|test\/fuzz\/\S+)
```

Comment:
Maybe add a comment to the build/codeql config files saying: "If changed, also change ... in `.mergify.yml`"? That way, if there's any need to update the paths in the future, these regexes won't get out of sync.

Reply:
Addressed in https://github.com/redhat-plumbers/systemd-rhel9/pull/161/commits/55c11ce8fd7ad9ce46d212504ed2262db2fee454